Earthfile

```diff
@@ -93,6 +95,27 @@ lint-newline:
         done; \
         exit $code
 
+update-dist-for-renovate:
+    FROM alpine/git
+    RUN git config --global user.name "renovate[bot]" && \
+        git config --global user.email "renovate[bot]@users.noreply.github.com" && \
```

Comment:
lol..... I guess this is the reason PGP commit signing exists :)

Reply:
For context - the only reason to do this is because by default when pushing changes to a Renovate branch, it decides to abandon the branch so it doesn't get updated.
For example, by default, if a PR is opened in order to update some dependency from v1.0.0 to v1.0.1 and while the PR stays open, v1.0.2 becomes available, Renovate will update the PR to try to update to the latest available version. So using the user name and email lets Renovate continue managing it.